src/component/SelectSearch/SelectSearch.tsx

```diff
@@ -66,45 +66,18 @@ export default class SelectSearch extends React.PureComponent<
         }
     }
 
-    private onChange(selection: Asset): void {
-        // Account for null selections
-        if (!selection) {
-            return;
-        }
-
-        if (selection.isNew && this.props.assets) {
-            this.props.assets.add(selection);
-        }
-
-        // Convert to array to update state
-        const selections = [selection];
-
-        if (!isEqual(this.state.selections, selections)) {
-            if (this.props.onChange) {
-                this.props.onChange(selection);
-            }
-
-            this.setState(
-                {
-                    selections
-                },
-                () => this.select.focus()
-            );
-        }
```

Comment:
instead have a shared implementation with the multi case

Reply:
Oh sweet, that's a win for sure.